spec/factories/whitehall_export/index_document_factory.rb

```diff
@@ -4,13 +4,10 @@
 
     sequence(:document_id)
 
-    document_information {
-      {
-        locales: %w(en),
-        sub_types: %w(news_story),
-        lead_organisations: %w(96ae61d6-c2a1-48cb-8e67-da9d105ae381),
-      }.stringify_keys
-    }
+    content_id { SecureRandom.uuid }
+    locales { %w(en) }
```

Comment:
Since they are not being exported from Whitehall now, `locales`, `sub_types` and `lead_organisations` can all be removed from this factory.

Reply:
👍